src/features/movie/components/imageContainer/ImageContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { MovieProps } from "../../../../models/movie.model";
```

Comment:
This is an import with so many levels of indirection. It makes the code fragile - if you move the component, you must ensure all these relative imports are updated. This is very error-prone.

Check if you can configure the preset like this: 
https://create-react-app.dev/docs/importing-a-component/#absolute-imports

Also, keep the component prop definitions in a component and global type definitions in the root.

Reply:
Done

Reply:
Done,

**Also, keep the component prop definitions in a component and global type definitions in the root.**

I don't fully undarstand